EPRN.Portal/Profiles/PRNProfile.cs

```diff
@@ -31,6 +31,13 @@ public PRNProfile()
 
             CreateMap<PRNHistoryDto, PRNHistoryViewModel>();
             CreateMap<StatusAndProducerDto, CancelViewModel>();
+            CreateMap<PRNDetailsDto, DraftConfirmationViewModel>()
+                .ForMember(d => d.Reference, o => o.MapFrom(s => s.ReferenceNumber))
+                .ForMember(d => d.DoWithPRN, o => o.MapFrom(s =>
```

Comment:
Shouldn't the most recent status be retrieved at the API Iside, rather than retrieve all of them and filter them out at the Portal side? Seems inefficient to me

Reply:
Done.